src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/IssueNumberRetriever.java

```diff
@@ -23,7 +27,18 @@ public IssueNumberRetriever(DataSource dataSource) {
     public long getNextIssueNumber() {
         try {
             Statement statement = dataSource.getConnection().createStatement();
-            ResultSet resultSet = statement.executeQuery(GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE);
+            ResultSet resultSet = statement.executeQuery(GET_NEXT_VALUE_FROM_ISSUE_NUMBER_SEQUENCE);
```

Comment:
Why not using Spring [JdbcTemplate](https://docs.spring.io/spring-framework/docs/current/spring-framework-reference/html/jdbc.html) then and let them manage closing the connections, statements and resultsets? That may help us avoid such a issues in the future.